src/Simulation/Simulators.Tests/StackTraceTests.cs

```diff
@@ -53,7 +53,12 @@ public void AllocateQubit2Test()
                 Assert.Equal(OperationFunctor.Body, stackFrames[0].Callable.Variant);
                 Assert.Equal(OperationFunctor.Body, stackFrames[1].Callable.Variant);
 
-                Assert.Equal(94, stackFrames[1].FailedLineNumber);
+                // TODO: The following change -
```

Comment:
It would be good to address this before merging.

Reply:
#678.
@DmitryVasilevsky, FYI.